### PR TITLE
refactor(suites): prevent log truncation in buildkite

### PR DESCRIPTION
**Pull request by nightah**

### CODE REVIEW

**Review by james-d-elliott — APPROVED** [review]
LGTM

File: internal/suites/docker.go [hunk 1]
```diff
@@ -52,6 +52,10 @@ func (de *DockerEnvironment) Pull(images ...string) error {
 
 // Up spawn a docker environment.
 func (de *DockerEnvironment) Up() error {
+	if os.Getenv("CI") == t {
+		return de.createCommandWithStdout("up --build --quiet-pull -d").Run()
+	}
+
```
[the remark this answers is not in the document]
Comment (james-d-elliott) [reply internal/suites/docker.go:58]: I pity the fool who don't check consts.go, go check there, sucka!

[the remark this answers is not in the document]
Comment (nightah) [reply internal/suites/docker.go:58]: You can see this defined https://github.com/authelia/authelia/blob/master/internal/suites/const.go#L89.

**Comment by james-d-elliott** [comment]
@coderabbit suggest a complete commit message for this PR which is going to be squashed.

**Pull request merged**